cltk/phonology/syllabify.py

```diff
@@ -0,0 +1,168 @@
+__author__ = ['Eleftheria Chatziargyriou <ele.hatzy@gmail.com>']
+__license__ = 'MIT License. See LICENSE.'
+
+import logging
+from cltk.exceptions import InputError
+
+LOG = logging.getLogger(__name__)
+LOG.addHandler(logging.NullHandler())
+
+
+class Syllabifier:
+
+    def __init__(self, low_vowels=[], mid_vowels=[], high_vowels=[], flaps=[], laterals=[], nasals=[], fricatives=[],
```

Comment:
Please correct, see: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Fixed this! I think using inline if expression is better than modifying locals or something equally hackish.